torch/fx/experimental/unify_refinements.py

```diff
@@ -20,8 +20,16 @@ def infer_symbolic_types(traced):
     to infer all the information such as the case
     for braodcasting.
     """
-    infer_symbolic_types_single_pass(traced)
-    infer_symbolic_types_single_pass(traced)
```

Comment:
Is `infer_symbolic_types_single_pass` still used, or can it be removed?

Reply:
I figured it should be there if someone wants to run a single pass? There is a case with broadcasting where doing two passes is needed. I will remove it for now since it would be easy to reconstruct it.